restli-tools/src/test/extensions/invalidExtensionAnnotation/FooExtensions.pdl

```diff
@@ -0,0 +1,4 @@
+record FooExtensions includes Foo {
```

Comment:
If we can add a comment say this is valid or not valid, then it is better, 
or we can use the name of the extension schema directly to tell it is valid or not valid.

Reply:
I see you put all invalid in one directory, so for my prev comment it is fine,
but if you can add some reason in the comment say why it is not valid. Then it is perfect.

Reply:
Added comment docs with more details.